### DISCUSSION TITLE
articles/why-making-a-starter-template-matters

### DISCUSSION THREAD

**Discussion by giscus** [yehezkielgunawan/yehezgun-new#13] [Article Post]
# articles/why-making-a-starter-template-matters

Yehezkiel Gunawan's Article Post

https://yehezgun.com/articles/why-making-a-starter-template-matters

**Comment by yehezkielgunawan** [comment]
### 👋 Hi there! Hope you're all good.
You can discuss about this topic here or share this readings if you like it 😁
